proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -44,157 +44,127 @@ direction of the policies set by the Board.
-   * approving new projects within the scope for _$FOUNDATION_ set by the Governing Board,
-     as outlined in [XXXX], and creating conceptual architecture for how the proejcts
-     fit together.
-   * aligning projects, removing or archiving projects, as outlined in [XXXX].
-   * approving funds for budgets delegated to the project. A good example is the
-     travel fund.
-   * follow and be up to date on Board/$FOUNDATION initiatives and communicate them to the projects.
-   * defining common practices to be implemented across _$FOUNDATION_ projects, if any.
-   * The CPC is also responsible for mediating technical conflicts between
-     _$FOUNDATION_ Projects when attempts to resolve those conflicts within the Project
-   were unsuccessful. 
-   * The CPC serves as the _$FOUNDATION_'s primary technical
-     liaison body with external open source projects, consortiums and groups and is
-     also responsible for ensuring that the _$FOUNDATION_ has appropriate technical
-     participation in standards bodies by finding an encouraging projects members to
-     particiapte in these bodies.
-   * Members of the CPC, or committees consisting of subsets of the CPC
-     membership will be expected to meet on a regular basis to discuss topics
-     such as new project acceptance into the mentorship program, project graduation
-     from the mentorship program, Project issues and conflicts, opportunities
-     for collaboration between Projects, opportunities for the Foundation in the
-     greater JavaScript community, etc.
+   1. defining and maintaining neutral consensus for the technical vision for the _$FOUNDATION_.
+   1. creating conceptual architecture for how the projects fit together.
+   1. Voting members are responsible for approving new projects within the scope
+      for _$FOUNDATION_ set by the Governing Board, as outlined in [XXXX].
+   1. Voting members are responsible for making final decisions on aligning projects,
+      removing or archiving projects, as outlined in [XXXX].
+   1. Voting members are responsible for approving funds for budgets delegated 
+      to the project. A good example is the travel fund.
+   1. follow and be up to date on Board/$FOUNDATION initiatives and communicate them to the projects.
+   1. defining common practices to be implemented across _$FOUNDATION_ projects, if any.
+   1. The CPC is also responsible for mediating technical conflicts between
+      _$FOUNDATION_ Projects when attempts to resolve those conflicts within the Project
+      were unsuccessful. 
+   1. Creating a yearly budget that can be drawn on by the projects.
+   1. The CPC serves as the _$FOUNDATION_'s primary technical
+      liaison body with external open source projects, consortiums and groups and is
+      also responsible for ensuring that the _$FOUNDATION_ has appropriate technical
+      participation in standards bodies by finding an encouraging projects members to
+      particiapte in these bodies.
```

Comment:
Does this impede on individual project's ability to participate in standards. This seems a bit out of scope to me... I was imagining that we would likely have a committee focused specifically on standards... being able to represent the view points of all members is a lot of work. This seems like something to delegate, but unsure if it should be chartered

Reply:
I think it was trying to say the CPC had the lead, not to participate in all of the bodies. "encouraging project members" is meant to be members from the projects who are affected by the standards bodies, not the members of the CPC.  The key question is whether we need the CPC to provide some leadership in pushing for participation or just leave it up to each of the individual projects to do what they think is best.  I'm thinking providing some leadership makes sense.